app/views/devise/sessions/new.html.erb

```diff
@@ -1,17 +1,77 @@
-<div class="page_header">
-  <div class="header_title">
-    <h3>WELCOME TO THE COMMON METADATA REPOSITORY (CMR) DASHBOARD <%= get_environment_display_name %></h3>
+<header class="landingpage-header" role="eui-banner">
+  <div class="content">
+    <div class="col-6">
+      <div class="eui-masthead-logo eui-application-logo" id="masthead-logo">
+        <a href="/">
+          <h1>CMR Metadata Curation Dashboard</h1>
+        </a>
+      </div>
+    </div>
+    <div class="col-6 header-tools">
+      <a href="#" id="help"><i class="fa fa-question-circle"></i>Help</a>
+
+      <% if devise_mapping.omniauthable? %>
+        <% resource_class.omniauth_providers.each do |provider| %>
+          <%= link_to "Login with #{get_omniauth_provider_display_name(provider)}", omniauth_authorize_path(resource_name, provider), class: "eui-btn--blue" %>
+        <% end %>
+      <% end %>
+    </div>
+  </div>
+</header>
+
+<noscript>
+  <div id='javascript_warning' class="eui-banner--danger">
+    <p class="eui-banner__message">
+      <strong> JavaScript is disabled! </strong> JavaScript must be enabled for some features to work.
```

Comment:
I feel like this belongs more in one of the layout files

Reply:
I made it a partial instead since it is used in two places:
```
<%= render(partial: 'shared/notice_message') %>
```